x-pack/plugins/cloud_security_posture/public/pages/configurations/utils/generate_findings_tags.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CspFinding } from '@kbn/cloud-security-posture-common';
```

Comment:
why this function file is added the function `generateFindingsTags` isn't used anywhere. It looks like you have a merge conflict with my clean up PR, some of the dead code which is not being used creeped in again